app/src/docs/v1.api-spec.yaml

```diff
@@ -1231,6 +1231,58 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /rbac/current/submissions:
+    get:
+      summary: Get submissions/roles/permissions for current user
+      operationId: getCurrentUserSubmissions
+      tags:
+        - RBAC
+      parameters:
+        - in: query
+          name: formId
+          schema:
+            type: string
+            format: uuid
+            example: c6455376-382c-439d-a811-0381a012d696
+          description: form ID
+        - in: query
+          name: formSubmissionId
+          schema:
+            type: string
+            format: uuid
+            example: c6455376-382c-439d-a811-0381a012d696
+          description: submission ID
+        - in: query
+          name: userId
+          schema:
+            type: string
+            format: uuid
+            example: c6455376-382c-439d-a811-0381a012d696
+          description: user ID
+        - in: query
+          name: active
+          schema:
+            type: boolean
+          example: true
+          description: to include active or deleted submissions or not
+      responses:
+        "200":
+          description: "OK"
+          content:
+            application/json:
+              schema:
+                allOf:
```

Comment:
Since we're not specifying multiple schema references, I don't think we need an allOf clause here and can just reference the UserSubmission schema object directly with the array.

Reply:
👍